cnxepub/tests/test_formatters.py

```diff
@@ -433,6 +433,10 @@ def test_binder(self):
             expected_content = f.read()
 
         actual = str(SingleHTMLFormatter(self.desserts))
+        # Uncomment this if/when you get a test failure
```

Comment:
This is the 4th time I've typed that from memory. Usually get it wrong at least once. :-/